python/understack-workflows/understack_workflows/main/nautobot_update_cf.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+
+from sushy.main import os
+
+from understack_workflows.helpers import arg_parser
+from understack_workflows.helpers import credential
+from understack_workflows.helpers import setup_logger
+from understack_workflows.nautobot import Nautobot
+
+
+def argument_parser():
+    parser = argparse.ArgumentParser(
+        prog=os.path.basename(__file__),
+        description="Ironic to Nautobot provisioning state sync",
+    )
+    parser.add_argument("--device_uuid", required=True, help="Nautobot device UUID")
```

Comment:
We can do this as a follow up but I think we should probably standardize this arg. The other entry points use `--device-id` and they use `type=UUID` to validate the input